src/domain/session/room/timeline/tiles/SimpleTile.js

```diff
@@ -129,6 +129,10 @@ export class SimpleTile extends ViewModel {
         return this._options.room;
```

Comment:
can we make RoomViewModel._room public, change this to _roomVM.room and not pass the room in and have one option less?